src/main/java/org/team1540/robot2018/Robot.java

```diff
@@ -78,6 +88,28 @@ public void robotInit() {
     zeroElevator.setRunWhenDisabled(true);
     SmartDashboard.putData(zeroElevator);
 
+    new Thread(() -> {
+      UsbCamera camera = CameraServer.getInstance().startAutomaticCapture(Tuning.camID);
+      camera.setResolution(640, 480);
+
+      CvSink cvSink = CameraServer.getInstance().getVideo();
+      CvSource outputStream = CameraServer.getInstance().putVideo("Camera "+Tuning.camID, 640, 480);
+
+      Mat source = new Mat();
+      Mat output = new Mat();
+
+      while(!Thread.interrupted()) {
+        cvSink.grabFrame(source);
+        //				Imgproc.cvtColor(source, output, Imgproc.COLOR_BGR2GRAY);
+        Point pt1 = new Point(source.width()/2+Tuning.crosshairsSize, source.height()/2);
+        Point pt2 = new Point(source.width()/2-Tuning.crosshairsSize, source.height()/2);
+        Point pt3 = new Point(source.width()/2, source.height()/2+Tuning.crosshairsSize);
+        Point pt4 = new Point(source.width()/2, source.height()/2-Tuning.crosshairsSize);
+        Imgproc.line(source, pt1, pt2, new Scalar(0,255,0), Tuning.crosshairsThicccness);
+        Imgproc.line(source, pt3, pt4, new Scalar(0,255,0), Tuning.crosshairsThicccness);
+        outputStream.putFrame(source);
+      }
+    }).start();
```

Comment:
crosshair code is useful but shouldn't be added to the intake camera

Reply:
oh right i'm stupid

Reply:
if we aren't using a turret at all we don't need this